.travis.yml

```diff
@@ -0,0 +1,51 @@
+# The language in this case has no bearing - we are going to be making use of conda for a
+# python distribution for the scientific python stack.
+language: generic
+
+sudo: false
+
+env:
+    global:
+       - CONDA_INSTALL_LOCN="${HOME}/conda"
+
+       # Adds a GH_TOKEN (conda-forge-admin's conda-forge.github.io token)
+       - secure: __FIX_ME__
```

Comment:
When you have a chance @pelson, can you please let me know what the encrypted string is for this. Should be for @conda-forge-admin I believe.